### DISCUSSION TITLE
RuntimeError: Unrecognized CachingAllocator option: max_split_size_mb=8000

### DISCUSSION THREAD

**Discussion by Eikewi** [oobabooga/text-generation-webui#929] [Q&A]
Hey,
i have a rtx 3070 ti with 8 gb vram and i tried to just run the WebUI but i always run out of memory before i even can choose which model i want to use. I tried multiple starting options like: --auto-devices --chat --wbits 4 --groupsize 128 --load-in-8bit --no-cache --disk --gpu-memory 8, but nothing seems to work. Has anyone a tip for me?
![Screenshot_2023-04-08_233547](https://user-images.githubusercontent.com/107319316/230769899-3a060055-9347-4001-9f6e-44784713f071.png)

**Comment by Wolvensdale** [comment]
You need to read and learn more about each parameters.
There are many unwritten rules. 
For example some parameters can't be combined with others. Like if you use autodevices its automatically wont use 8bit,
Or when you use wbits, then its automatically assume that the model you used are quantized one, therefore all other command that are for other model (like ggml, and normal one) won't work. cmiiw.

**Reply by adollar47** [reply]
Please share your parameters if possible. I'm new to all this and need a place to start from

**Reply by Eikewi** [reply]
I tried many combinations of commands, but nothing seems to work. I am not even sure if memory is even the problem, because it should ran with the specs i have. It seems more like a problem during Setup. I also search the web for the error but i didn't find anything helpful. Even gpt4 cant help me.

**Comment by Besamel** [comment]
I got it working with the following options: --auto-devices --chat --model gpt4-x-alpaca-13b-native-4bit-128g --wbits 4 --groupsize 128 --gpu-memory 5 --pre_layer 20

It's a bit slow and dumb though

**Reply by TheBlackTesla** [reply]
How did you change it those options?

**Reply by Besamel** [reply]
Edit start-webui.bat in a text editor

**Reply by Eikewi** [reply]
Didn´t work for me either. It seems like it is already failing at the SETUP so maybe it isn´t something about the memory. I uploaded a screenshot of the error maybe it helps understanding the problem.

**Reply by Besamel** [reply]
Try restarting your machine and trying to launch it before opening anything else. I had no luck before I restarted my machine.

**Reply by Eikewi** [reply]
Tried it and i tried to reinstall it from scratch, but didnt work either ):

**Reply by elcotek** [reply]
I use the RTX 2080 8GBvram and `--auto-devices --disk --groupsize 128 --load-in-8bit --gpu-memory 3724MiB --pre_layer 25 --chat`  - Right, i use only 3,724 GB of Vram for the chavinlo_alpaca-native model.

**Reply by Eikewi** [reply]
Didn´t work either
